kernel/packages/unity-interface/dcl.ts

```diff
@@ -134,14 +137,17 @@ export async function initializeEngine(_gameInstance: GameInstance) {
   }
 }
 
-export async function startGlobalScene(unityInterface: UnityInterface) {
-  const sceneId = 'dcl-ui-scene'
-
+export async function startGlobalScene(
+  unityInterface: UnityInterface,
+  cid: string,
+  title: string,
+  fileContent: string
```

Comment:
Could we make it clearer that this is a url?

Reply:
Done